tests/common.go

```diff
@@ -9200,6 +9193,39 @@ func GetClusterProvisionStatus() ([]ProvisionStatus, error) {
 			}
 		}
 	}
+	log.Infof("Cluster provision status [%v]", clusterProvision)
+	return clusterProvision, nil
+}
+
+// GetClusterProvisionStatus  returns details of cluster provision status
+func GetClusterProvisionStatus() ([]ProvisionStatus, error) {
+	// Using Node which is up and running
+	var selectedNode []node.Node
+	for _, eachNode := range node.GetStorageDriverNodes() {
+		status, err := IsPxRunningOnNode(&eachNode)
+		if err != nil {
+			log.InfoD("Px is not running on the Node.. searching for other node")
+			continue
+		}
+		if status {
+			selectedNode = append(selectedNode, eachNode)
+			break
+		}
+	}
+	if len(selectedNode) == 0 {
+		return nil, fmt.Errorf("No Valid node exists")
+	}
+
+	// Select Random Volumes for pool Expand
+	randomIndex := rand.Intn(len(selectedNode))
+	randomNode := selectedNode[randomIndex]
+
+	clusterProvision, err := GetClusterProvisionStatusOnSpecificNode(randomNode)
+	if err != nil {
+		return nil, err
+	}
+
+	log.Infof("Cluster provision status [%v]", clusterProvision)
```

Comment:
this is getting logged twice once here and once in  GetClusterProvisionStatusOnSpecificNode